readme.md

```diff
@@ -34,9 +34,14 @@ run the program with the following options (the default zig install directory is
    - `-n, --invert_color`: Inverts the color values (optional)
    - `-s, --scale <float>`: set the downscale or upscale factor (optional, default: 1)
    - `-e, --detect_edges`: enable edge detection (optional)
-   - `--sigma1 <float>`: set the sigma1 value for DoG filter (optional, default: 0.3)
-   - `--sigma2 <float>`: set the sigma2 value for DoG filter (optional, default: 1.0)
+   - `    --sigma1 <float>`: set the sigma1 value for DoG filter (optional, default: 0.3)
+   - `    --sigma2 <float>`: set the sigma2 value for DoG filter (optional, default: 1.0)
    - `-b, --brightness_boost <float>`: increase/decrease perceived brightness (optional, default: 1.0)
+   - `    --full_characters`: Uses full spectrum of characters in image.
+   - `    --ascii_chars <string>`: Use what characters you want to use in the image. (default: " .:-=+*%@#")
+   - `    --sorted_ovr`: Prevents sorting of the ascii_chars by size.
+   - `    --char_size <u8>`: Set the size of the characters. (default: 8)
```

Comment:
This doesn't actually set the size of the characters. But the size of each block of image considered. It's kind of pointless changing this. 

See this image below: This is what happens if you set it to a value that isn't divisible by 8. 

![CleanShot 2024-09-02 at 23 49 08@2x](https://github.com/user-attachments/assets/5a6df04b-8b67-4796-91a0-bd53880fcc4a)

And when the value is set to something high like 24, the image loses becomes blocky:
![image](https://github.com/user-attachments/assets/a47d0fa8-7bda-4f51-bc97-a88a633c835e)